Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -38,7 +38,7 @@ fileprivate let scaleOutTransformRight = CGAffineTransform.identity.translatedBy
  **/
 class PlaceDetailViewController: UIViewController {
 
-    weak var dataSource: PlaceDataSource? {
+    weak var dataSource: PlacesProvider? {
```

Comment:
Since we already have the `PlaceProvider` interface used for `Place`s, do you think it would make sense to keep the `PlaceDataSource` name here instead of `PlacesProvider`? Alternatively, we could find a new name for `PlaceProvider`.